src/unix/fs.c

```diff
@@ -84,7 +84,8 @@
 
 #if defined(__CYGWIN__) ||                                                    \
     (defined(__HAIKU__) && B_HAIKU_VERSION < B_HAIKU_VERSION_1_PRE_BETA_5) || \
-    (defined(__sun) && !defined(__illumos__))
+    (defined(__sun) && !defined(__illumos__)) ||                              \
+    (defined(__APPLE__) && (MAC_OS_X_VERSION_MAX_ALLOWED <= 101300))
```

Comment:
Style/consistency with line 86:
```suggestion
    (defined(__APPLE__) && MAC_OS_X_VERSION_MAX_ALLOWED <= 101300)
```